packages/billing/stripe-webhook-handlers/index.ts

```diff
@@ -149,5 +150,6 @@ export default async function stripeWebhookHandler(
       break;
   }
 
+  sendBillingNotification(event.data.object.customer);
```

Comment:
Stripe could be sending many different webhooks which we won't care about (we only care about the ones that are handled by our 3 handlers). So I think this will send a billing notification for stripe webhooks that we do not handle. I think that's redundant.

Reply:
I moved this line to after calling each handler.